docs: document missing options (#110)
developit/workerize-loader: README.md

import worker from 'workerize-loader?inline!./worker'
```

#### `name`

Type: `String`
Default: `[hash]`

> You can pass filename prefix for the compiled worker. Note, that `.worker` suffix will be added automatically

Reviewer: ```suggestion
Customize filename generation for worker bundles. Note that a `.worker` suffix will be injected automatically (`{name}.worker.js`).
```

```js
// webpack.config.js
{
  loader: 'workerize-loader',
  options: { name: '[name].[contenthash:8]' }
}
```
or 
```js
import worker from 'workerize-loader?{"name":"[name].[contenthash:8]"}!./worker'
```

#### `publicPath`

Type: `String`
Default: based on `output.publicPath`

> You can pass `publicPath` to specify the public URL address of the output files

Reviewer: ```suggestion
Workerize uses the configured value of `output.publicPath` from Webpack unless specified here. The value of `publicPath` gets prepended to bundle filenames to get their full URL. It can be a path, or a full URL with host.
```

```js
// webpack.config.js
{
  loader: 'workerize-loader',
  options: { publicPath: '/static/' }
}
```
or 
```js
import worker from 'workerize-loader?{"publicPath":"/static/"}!./worker'
```

#### `ready`

Type: `Boolean`
Default: `false`

> You can pass `ready` to set up a wrapper around the `"ready"` event

Reviewer: ```suggestion
If `true`, the imported "workerized" module will include a `ready` property, which is a Promise that resolves once the Worker has been loaded. Note: this is unnecessary in most cases, since worker methods can be called prior to the worker being loaded.
```

```js
// webpack.config.js
{
  loader: 'workerize-loader',
  options: { ready: true }
}
```
or 
```js
import worker from 'workerize-loader?ready!./worker'

let instance = worker()  // `new` is optional

await instance.ready
```

> #### `fallback`
> 
> Type: `Boolean`
> Default: `true`
> 
> // TODO: document fallback option
> 
> ```js
> // webpack.config.js
> {
>   loader: 'workerize-loader',
>   options: { fallback: false }
> }
> ```
> or 
> ```js
> import worker from 'workerize-loader?fallback=false!./worker'
> ```
> 

Reviewer: I have no idea what the fallback option is supposed to be for 😅

For now let's leave it undocumented since it's disabled by default and should probably be removed.

```suggestion
```

#### `import`

Type: `Boolean`
Default: `false`

> // TODO: document import option

Reviewer: ```suggestion
When enabled, generated output will create your Workers using a Data URL that loads your code via `importScripts` (eg: `new Worker('data:,importScripts("url")')`). This workaround enables cross-origin script preloading, but Workers are created on an "opaque origin" and cannot access resources on the origin of their host page without CORS enabled. Only enable it if you understand this and specifically need the workaround.
```
